src/Listener/LogRequest.php

```diff
@@ -40,8 +41,8 @@ public function onRoute(MvcEvent $e)
     {
         if ($this->isConsole($e)) {
             $data = [
-                'path' => $e->getRequest()->getScriptName(),
-                'params' => $e->getRequest()->getParams()
+                'path' => $_SERVER['argv'][0],
+                'params' => array_slice($_SERVER['argv'], 1)
```

Comment:
Can we put these to come from methods via. the interface instead, please?

Reply:
Done